packages/core/src/components/Tooltip/index.tsx

```diff
@@ -190,8 +194,34 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
       inverted,
     } = this.props;
     const { open, targetRect, tooltipHeight, labelID } = this.state;
+
+    if (!open) {
+      return (
+        <span ref={this.containerRef} className={cx(styles.container)}>
+          <div
+            aria-labelledby={labelID}
+            className={cx(!disabled && underlined && styles.underlined)}
+            onMouseEnter={this.handleEnter}
+            onMouseLeave={this.handleClose}
+            onMouseDown={this.handleMouseDown}
+          >
+            {children}
+          </div>
+
+          {/* render off-screen element in a separate layer */}
+          <Portal>
+            <div id={labelID} className={cx(styles.offscreen)}>
+              {content}
+            </div>
+          </Portal>
+        </span>
+      );
```

Comment:
This appears to duplicate some stuff. It would be great to do this in a way that avoids the duplication, to keep the size of the JS smaller, make it easier to maintain, and prevent the two versions from drifting.